readme.md

```diff
@@ -0,0 +1,184 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+<!--   <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/> -->
+
+<!--   <h3><b>Microverse README Template</b></h3> -->
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 🔬 Awesome-books<a name="about-project"></a>
+
+This project is for an list of awesome books in a webpage
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+  </ul>
+  <ul>
+    <li>JavaScript</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- List structure
+  - Set up a new project with webpack
+  - The list appear in order of the index values for each task.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: any functional browser.
+
+### Setup
+
+Clone this repository to your desired folder:
+cd my-folder git clone [@fithamlak](https://github.com/fithamlak/To-do-list)
+
+### Install
+
+Install this project with 
+
+### Usage
+
+To run the project, execute the following command:
+open your live server
+
```

Comment:
- [ ] The project uses webpack, there are some commands we run to have the project ready on the local machine and show in the browser. Kindly list all the commands users would use to run the project.